.coderabbit.yaml

```
@@ -1,14 +1,94 @@
+  3. 비판보다는 개선 중심의 제안을 우선하세요.
+  4. 칭찬은 짧고 위트 있게 작성하세요.
+
+# ---------------------------------------------------------------- #
+# 코드 리뷰 기능 세부 설정
+# ---------------------------------------------------------------- #
 reviews:
-  profile: "chill" # 리뷰 너무 빡세게는 안한다는 뜻
-  request_changes_workflow: true # 코드래빗이 리뷰 끝나면 알아서 PR 승인하게끔
+  # 리뷰의 전반적인 스타일 - chill: 덜 엄격, 부드러운 톤
+  profile: "chill"
+  # CodeRabbit의 코멘트를 모두 `resolve`할 경우, 자동 Approve 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)